chart/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -38,6 +38,7 @@ resources: {}
 provisioner:
   URL: "http://compass-provisioner.compass-system.svc.cluster.local:3000/graphql"
   dumpRequests: "false"
+  timeout: "12h"
```

Comment:
hm.. I was thought that here we are storing values directly related to "provisioner" component. This timeout is not sent to provisioner it's only about our timeout per step, right? Provisioner for installation has it own timeout https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/provisioner/templates/deployment.yaml#L81-L82

For me it's quite confusing.

btw. should we also take care of the installation timeout on provisoner side? because having 24h per step in our KEB is not enough

Reply:
it is about provisioner! it is how long should we check the status in the provisioner.

Reply:
there will be (or it is merged) also change in the provisioner

Reply:
IMO we should add a comment above this timeout field to explicitly said that it is not send to provisoner and provisoner has it’s own timeout which should also be taken into account because if I will not know the provisioner part then I will thought that it is about provionser timeout. Just check the name ‘proviosner.timeout’ - it is really confusing